src/js/gtm-ga.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+// Needs dnt-helper.js to be executed first and window.Mozilla.googleTagId to be set
+
+(function () {
+    "use strict";
+
+    // dnt-helper.js sets the window.Mozilla namespace
+    var GOOGLE_TAG_ID = window.Mozilla.googleTagId;
```

Comment:
Yeah it would - better to deal with all of it in one single file, then if it fails to load, we get no GA, but that's better than failing to respect DNT